gtk2_ardour/add_route_dialog.h

```diff
@@ -59,7 +59,7 @@ class AddRouteDialog : public ArdourDialog
 {
 public:
 	AddRouteDialog ();
-	~AddRouteDialog ();
+	~AddRouteDialog () override;
```

Comment:
Is there a reason to keep this? The d'tor does nothing, so we should just remove it. Fewer LOC is better :)